### PR TITLE
Am branch

### PR DESCRIPTION
**Pull request by carlo-activamedia**

### CODE REVIEW

**Comment by xkjyeah** [comment]
Hi, could you merge in the latest version first? There are merge conflicts, unfortunately

File: app.html [hunk 1]
```diff
@@ -1,28 +1,38 @@
...
+    </script>
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-115289012-1"></script>
+    <script>
+      
+    window.dataLayer = window.dataLayer || [];
+    function gtag(){dataLayer.push(arguments);}
+    gtag('js', new Date());
+    gtag('config', 'AW-952434988');
+    </script>
```
Comment (danielsim-stripe) [inline app.html]: I think this is a duplicate gtag. you might want to delete this section

Comment (xkjyeah) [inline app.html]: i believe this is a duplicate section. Could you delete lines 13-21?

**Comment by carlo-activamedia** [comment]
**correction:** the website should have 2 google analytic codes.. 
the existing one AW-952434988 and our google analytic code which is UA-115289012-1..
I guess, I accidentally replaced the existing code with ours and became duplicate.

**Comment by xkjyeah** [comment]
Oh don't worry -- you didn't delete it. I deleted it.
The problem is that there seems to be a naming conflict -- both pieces of code define `gtag`. So to avoid conflict, I deleted my tracking code.
Instead, you can share your tracking dashboard with me.

**Comment by carlo-activamedia** [comment]
pardon, what do you mean by sharing of tracking dashboard?

**Comment by xkjyeah** [comment]
On the Google analytics console, you can add a collaborator:

Screenshot 2020-02-18 at 13 15 22

**Comment by xkjyeah** [comment]
Meanwhile, please delete the code for  AW-952434988  -- it's not going to work if you install Google analytics for UA-115289012-1

**Comment by carlo-activamedia** [comment]
sorry for the delay. the AW-952434988 has been removed

**Pull request merged**